.code-samples.meilisearch.yaml

```diff
@@ -133,7 +133,7 @@ async_guide_filter_by_ids_1: |-
     }
   }
 async_guide_filter_by_statuses_1: |-
-  client.getTasks(params: TasksQuery(statuses: ["failed", "canceled"])) { result in
+  client.getTasks(params: TasksQuery(statuses: [.failed, .canceled])) { result in
```

Comment:
Is this PR breaking so?
Not an issue for me to break, we are not stable yet. It's to apply the right versioning and provide a clear changelog

Reply:
This is a breaking change, yes. The changes have been documented in the initial PR description, in case you want to copy them into a changelog 😀

Reply:
For what it's worth, and looking at tickets in the backlog, I'm intentionally looking to prioritise breaking changes now in order to get us to a stable position quicker.

Reply:
Ok let's merge it with breaking then! thank you for the help of the changelog ❤️